server/etcd_quota_provider.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/google/trillian/quota"
+	"github.com/google/trillian/quota/cacheqm"
+	"github.com/google/trillian/quota/etcd/etcdqm"
+	"github.com/google/trillian/util/etcd"
+)
+
+// QuotaEtcd represents the etcd quota implementation.
+const QuotaEtcd = "etcd"
+
+var (
+	// EtcdServers is a flag containing the address(es) of etcd servers
+	EtcdServers       = flag.String("etcd_servers", "", "A comma-separated list of etcd servers; no etcd registration if empty")
+	quotaMinBatchSize = flag.Int("quota_min_batch_size", cacheqm.DefaultMinBatchSize, "Minimum number of tokens to request from the quota system. "+
+		"Zero or lower means disabled. Applicable for etcd quotas.")
+	quotaMaxCacheEntries = flag.Int("quota_max_cache_entries", cacheqm.DefaultMaxCacheEntries, "Max number of quota specs in the quota cache. "+
+		"Zero or lower means disabled. Applicable for etcd quotas.")
```

Comment:
As in above comment zero or lower means batching disabled not quota disabled? Could be worth clarifying.

Reply:
I have no idea, I just moved the flags and code around - I didn't write this :)

Reply:
We'll take that under advisement M'Lud. I think adding "batching is" before disabled to both comments fixes it.

Reply:
Done :)